src/frontend/src/components/CollapsibleInstructions/style.ts

```diff
@@ -30,6 +29,37 @@ export const HeaderWrapper = styled("div", {
   }}
 `;
 
+export const InstructionsTitle = styled("div", {
```

Comment:
This change will modify all existing instructions titles. is that intentional?

Reply:
no it isn't, i updated